docs/docs/js/index.md

```
@@ -67,23 +66,48 @@ import TabItem from "@theme/TabItem";
-```html title="Paste this in your HTML file"
-<script type="module">
-  import { imageToUint8ClampedArray, bilateralFilter, kmeans, findContours } from "https://cdn.jsdelivr.net/npm/img2num/build-wasm/index.js";
+<Tabs defaultValue="jsDelivr">
+  <TabItem value="jsDelivr">
 
-  // Your code here
-</script>
-```
+  [![CDN: jsDelivr](https://img.shields.io/badge/CDN-jsDelivr-%23f7df1e?logo=jsdelivr&logoColor=black)](https://www.jsdelivr.com/package/npm/img2num)
+
+  ```html title="Paste this into your HTML file"
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)